plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -325,7 +329,11 @@ class NewServiceFormModal extends Component {
         {
           cmd: !isPod,
           container: !isPod,
-          containers: isPod
+          containers: isPod,
```

Comment:
can we split this into 2 lists: `PodReducers` `ServiceReducers`? Because now containers is a pod reducer surrounded by non-pod reducers and I didn't catch that right away.